freedomofpress/qubes-template-securedrop-workstation
# Update keys and URL for beta release

Use production endpoints, see https://github.com/freedomofpress/securedrop-workstation/pull/439 for test plan. 
### Important merge steps
#### Prior to merge
- [x] drop 1c7e11f (which will default to master for build)
#### Post merge
- [ ] Push a tag signature on the merge commit, with the release signing key

### Test Plan
#### Test the build script
- The build script was transfered from securedrop-workstation, see https://github.com/freedomofpress/securedrop-workstation/pull/439
- Compare diff of this branch and test-build-script branch: https://github.com/freedomofpress/qubes-template-securedrop-workstation/compare/beta-release..test-build-script
- check out test-build-script branch
- make template
- [x] Template build is successful

#### Test the template build with prod prior to merge
- [x] https://github.com/freedomofpress/securedrop-debian-packages-lfs/pull/9 is merged, and changes deployed
-  Replace the SD signing key fingerprint with mine:
```
diff --git a/build-workstation-template b/build-workstation-template
index e280461..6f2fac9 100755
--- a/build-workstation-template
+++ b/build-workstation-template
@@ -36,8 +36,8 @@ make COMPONENTS="builder" get-sources
 # Add signing key to the keyring. The pubkey fingerprint tracked here is the "FPF Authority Key".
 # In order to test PRs, you may want to use a personal key with a temporary signed tag.
 # See comments above regarding feature branches in the git clone operation.
-gpg --homedir ${gpg_homedir} --keyserver pool.sks-keyservers.net --recv-key 22245C81E3BAEB4138B36061310F561200F4AD77 || exit 1;
-echo '22245C81E3BAEB4138B36061310F561200F4AD77:6:' | gpg --homedir ${gpg_homedir} --import-ownertrust;
+gpg --homedir ${gpg_homedir} --keyserver pool.sks-keyservers.net --recv-key AF775782949D263DAABB3387AAFB3575FAC82745 || exit 1;
+echo 'AF775782949D263DAABB3387AAFB3575FAC82745:6:' | gpg --homedir ${gpg_homedir} --import-ownertrust;
 
 # Get all sources
 make get-sources
```
- check out this branch (beta-release)
- make template
- [x] template build is successful
